plenario/api/validator.py

```diff
@@ -263,13 +263,19 @@ def validate(validator, request_args):
             field = param.split('__')[0]
             if table is not None:
                 try:
-                    valid_column_condition(table, field, args[param])
+                    param = param.encode()
+                    value = args[param].encode()
```

Comment:
make an attempt to read the param and value that, if fails, lets us report to the user gracefully